docs/UserGuide.md

```diff
@@ -3,213 +3,268 @@ layout: page
 * [Features](https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html#features)
 * [FAQs](https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html#faq)
-* [Command Summary](https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html#command-summary)
+* [Command summary](https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html#command-summary)
 
 --------------------------------------------------------------------------------------------------------------------
 
-## [About this document](https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html)
+# Introduction
+
+SimplyKitchen is a desktop app for food inventory management, optimised for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). 
+With intuitive and practical features, it can get your food management tasks done faster and more efficiently than traditional GUI apps.
+
+SimplyKitchen aims to help you, a domestic household member, to manage your kitchen by providing a food inventory management system.
+We have taken into consideration the common problems you may face while managing your kitchen and have created specialised features to address them.
+
+Since the app is meant for the general households, care has been taken to make it more intuitive and not overwhelming to facilitate comfortable usage for all.
+We hope to make SimplyKitchen a household name in Singapore and appreciate your assistance in helping us do so!
+
+<div style="text-align: right"><a href="https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html">^ Back to top</a></div>
+
+--------------------------------------------------------------------------------------------------------------------
+
+# About this document
+
+This document is a User Guide meant to assist you in using SimplyKitchen to manage your food inventory.
 
-This document is a User Guide meant to assist you in the entire process of using SimplyKitchen to manage your food inventory.
+The **quick start** section guides you in setting up and launching the SimplyKitchen app on your computer.
 
-The `Quick start` section of this document guides you in setting up and launching the SimplyKitchen app on your computer.
+The **features** section helps you understand and use the features of SimplyKitchen. 
+It gives clear examples for each feature to ensure that the purpose and constraints of the feature are evident.
 
-The `Features` section of this document can help you understand and use the features of SimplyKitchen. 
-It gives clear examples for each of the features to ensure that the purpose and constraints of the features are evident.
+The **FAQs** section contains some Frequently Asked Questions with regard to SimplyKitchen.
 
-The `FAQs` section contains some Frequently Asked Questions with regard to SimplyKitchen.
+Finally, the **command summary** section acts as a quick reference point for all the features of SimplyKitchen.
```

Comment:
Do you think if we make \*\*command summary** a hyperlink it would be convenient? Same for FAQ, features and quick start above as well.

Reply:
Added hyperlinks. Thanks.